src/helpers/jwks.js

```diff
@@ -26,6 +26,10 @@ function getJWKS(options, cb) {
         return cb(err);
       }
 
+      if (!data.body) {
+        return cb({error: true}, data);
```

Comment:
Considering this error will be returned to the public API method (and then returned to auth0.js), we should have a more descriptive error message.

https://github.com/auth0/idtoken-verifier/blob/d30aa9310ac2f6a5ac1412a587ce062cc844013d/src/index.js#L201